site/content/en/docs/components/optional/gcsupload.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: "prow/cmd/gcsupload/README.md"
+title: "gcsupload"
```

Comment:
I am not sure whether `gcsupload` can be categorized into 'Optional Components' or not.